### DISCUSSION TITLE
苹果端 Todo App 简单比较 - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#13] [Announcements]
# 苹果端 Todo App 简单比较 - 1nan

https://yinan.me/todo-app.html

**Comment by versun** [comment]
哈哈，我使用的滴答，而且协作很方便，我目前正在测试它的笔记功能，准备也当作all in one了

**Reply by yinan-c** [reply]
嗯，滴答很可以，订阅费也不贵！算是全平台通用的为数不多的选择了。
